week-5-extended-project/team-space/db.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE tours (
+  id SERIAL PRIMARY KEY,
+  name varchar (100),
+  days integer,
+  price float(2)
+);
+
+CREATE TABLE customers (
+  id SERIAL PRIMARY KEY,
+  name varchar (100),
+  city varchar (100),
+  state varchar (100),
+  country varchar (100)
+);
+
+CREATE TABLE agencies (
+  id SERIAL PRIMARY KEY,
+  name varchar (100),
+  contact_name varchar (100)
+);
+
+CREATE TYPE payment_type AS ENUM ('Visa','Mastercard', 'Amex', 'Diners');
+CREATE TYPE pos_type AS ENUM ('Facebook','Whatsapp', 'Shop', 'Web');
```

Comment:
I like the way you have created these types

Reply:
Thanks, you can read more [here](https://www.postgresql.org/docs/10/datatype-enum.html)